fc/cli/reports.py

```diff
@@ -253,12 +211,17 @@ def vfrsanity(username: str):
 
     auth = ComboAuth(username)
 
-    SERVICE_ACCOUNT_FILE = auth.google_service_acct_creds()
-    SHEET_CREATE_URL = auth.sheet_create_url()
+    # setup and build credentials for google api calls
+    g_scopes = None
+    g_service_acct_file = None
+    g_sheet_create_url = None
```

Comment:
All the `None` assignments here can be removed since you are setting them just one line below.